sdcm/tester.py

```diff
@@ -866,19 +867,25 @@ def set_system_auth_rf(self, db_cluster=None):
             # TODO: move this skip to siren-tools when possible
             self.log.warning("Skipping this function due this job run from Siren cloud!")
             return
-        # change RF of system_auth
-        system_auth_rf = self.params.get('system_auth_rf')
-        if system_auth_rf > 1 and not self.test_config.REUSE_CLUSTER:
```

Comment:
You removed reused cluster support here.

Please put it back

Reply:
fixed